include/jemalloc/internal/prof_types.h

```diff
@@ -23,7 +23,8 @@ typedef struct prof_recent_s prof_recent_t;
  * is based on __builtin_return_address() necessarily has a hard-coded number
  * of backtrace frame handlers, and should be kept in sync with this setting.
  */
-#define PROF_BT_MAX			128
+#define PROF_BT_MAX_PROF_GCC        256
```

Comment:
Maybe do it this way to hide the PROF_GCC detail to non-prof modules:

```
#ifdef JEMALLOC_PROF_GCC
#  define PROF_BT_MAX_LIMIT 256
#else
#  define PROF_BT_MAX_LIMIT UINT_MAX
#endif
```

Then we can use `PROF_BT_MAX_LIMIT` anywhere and avoid `ifdefs` around it.